### ISSUE TITLE
Can't archive after install firebasex ( IOS ) Signature Error

### ISSUE THREAD

**Issue by MLuan** [dpa99c/cordova-plugin-firebasex#408]
# Bug report

**CHECKLIST**
- [x] I have read the [issue reporting guidelines](https://github.com/dpa99c/cordova-plugin-firebasex#reporting-issues)

- [x] I confirm this is a suspected bug or issue that will affect other users

- [x] I have reproduced the issue using the [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) or provided the necessary information to reproduce the issue.

- [x] I have read [the documentation](https://github.com/dpa99c/cordova-plugin-firebasex) thoroughly and it does not help solve my issue.

- [x] I have checked that no similar issues (open or closed) already exist.

**Current behavior:**
After install this plugin and archive, i can't upload that .ipa file to apple store because of the error above. The build is ok, i can run in my device but i can't upload to Apple store. 
I have another plugin that use cocoa pod and i don't have problema with upload. 

**Expected behavior:**

**Steps to reproduce:**

**Reproduce steps**
* Git clone [app test ionic 3](https://github.com/dpa99c/cordova-plugin-firebasex-ionic3-test)
* Follow the readme.md instructions
* Make a IOS build.
* Adjust signing code with the certificates and profiles
* Create a archive
* Click in "Validate"

**Screenshots**

![image](https://user-images.githubusercontent.com/22017044/82646341-72d99e00-9be2-11ea-8bf6-c33abcc0043e.png)

**Environment information**

- Cordova CLI version 
	- 9.0.0
- Cordova platform version
	- ios 5.1.1
- Plugins & versions installed in project (including this plugin)
    - com-badrit-base64 0.2.0 "Base64"
    - cordova-android-firebase-gradle-release 4.0.0 "cordova-android-firebase-gradle-release"
    - cordova-android-play-services-gradle-release 4.0.0 "cordova-android-play-services-gradle-release"
    - cordova-android-support-gradle-release 3.0.1 "cordova-android-support-gradle-release"
    - cordova-clipboard 1.3.0 "Clipboard"
    - cordova-plugin-add-swift-support 2.0.2 "AddSwiftSupport"
    - cordova-plugin-android-permissions 1.0.2 "Permissions"
    - cordova-plugin-androidx 1.0.2 "cordova-plugin-androidx"
    - cordova-plugin-androidx-adapter 1.1.1 "cordova-plugin-androidx-adapter"
    - cordova-plugin-app-version 0.1.9 "AppVersion"
    - cordova-plugin-browsertab 0.2.0 "cordova-plugin-browsertab"
    - cordova-plugin-camera 4.1.0 "Camera"
    - cordova-plugin-compat 1.2.0 "Compat"
    - cordova-plugin-contacts 2.3.1 "Contacts"
    - cordova-plugin-device 1.1.7 "Device"
    - cordova-plugin-facebook4 6.4.0 "Facebook Connect"
    - cordova-plugin-file 6.0.2 "File"
    - cordova-plugin-file-opener2 2.2.1 "File Opener2"
    - cordova-plugin-file-transfer 1.7.1 "File Transfer"
    - cordova-plugin-firebasex 9.1.1 "Google Firebase Plugin"
    - cordova-plugin-globalization 1.11.0 "Globalization"
    - cordova-plugin-inappbrowser 3.2.0 "InAppBrowser"
    - cordova-plugin-ionic-keyboard 2.2.0 "cordova-plugin-ionic-keyboard"
    - cordova-plugin-ionic-webview 4.2.1 "cordova-plugin-ionic-webview"
    - cordova-plugin-market 1.2.0 "Market"
    - cordova-plugin-network-information 1.3.4 "Network Information"
    - cordova-plugin-pagmovel 1.0.18-SNAPSHOT "PagMovelCordovaPlugin"
    - cordova-plugin-splashscreen 4.1.0 "Splashscreen"
    - cordova-plugin-statusbar 2.4.3 "StatusBar"
    - cordova-plugin-uniquedeviceid 1.3.2 "UniqueDeviceID"
    - cordova-plugin-whitelist 1.3.4 "Whitelist"
    - cordova-plugin-x-socialsharing 5.6.6 "SocialSharing"
    - es6-promise-plugin 4.2.2 "Promise"
    - ionic-plugin-deeplinks 1.0.20 "Ionic Deeplink Plugin"
    - phonegap-plugin-barcodescanner 8.1.0 "BarcodeScanner"
    - phonegap-plugin-mobile-accessibility 1.0.5-dev "Mobile Accessibility"
- Dev machine OS and version, e.g.
    - OSX
        - ProductName:	Mac OS X
        - ProductVersion:	10.15.1
        - BuildVersion:	19B88

_Runtime issue_
- Device details
    - iPhone 11
- OS details
    - iOS 11.0
	
_iOS build issue:_
- Node JS version
    - v12.16.3
- XCode version
    - 11.3.1 (11C504)

</details><br/><br/>

**Comment by dpa99c** [comment]
The error message shown in your screenshot is a signing error so not directly caused by this plugin.
Maybe this is because your app does not have a valid APNS certificate which is required since this plugin implements Push Notifications for Firebase Cloud Messaging?

This plugin is used in production by many other apps so this problem is specific to your configuration.

**Comment by MLuan** [comment]
I see, I used cordova-plugin-firebase, the certificates are correct. I had to migrate to this plugin because of the webview. I thought it would be something related to the plugin because without it I can do the complete process. And the facebook plugin for example also creates a PodFile and does not interfere with the build or archiving. I already checked the certificates, if they are correct all day, so I opened an Issue in the hope that someone has already faced something related.
But ok, thanks for your help.
